documentation/release_notes.rst

```diff
@@ -8,16 +8,53 @@ The Intel® oneAPI DPC++ Library (oneDPL) accompanies the Intel® oneAPI DPC++/C
 and provides high-productivity APIs aimed to minimize programming efforts of C++ developers
 creating efficient heterogeneous applications.
 
+New in 2021.6
+=============
+
+New Features
+------------
+- Added a new implementation for ``par`` and ``par_unseq`` execution policies based on OpenMP* 4.5 pragmas.
+  It can be enabled with the ``ONEDPL_USE_OPENMP_BACKEND`` macro. For more details, see `the Macros`_ page.
```

Comment:
Probably "see \_the Macros page\_ in the library guide" (because I think that release notes are a separate document, not a part of the guide)

Reply:
Added the mention of the library guide